float32[] TargetObjectFlatPointCloud
---
float32[] GraspingPose